src/data/common/entities/AuditAnalytics.ts

```diff
@@ -0,0 +1,11 @@
+export function getEventQueryString(
+    programId: string,
+    programStageId: string,
+    orgUnitIds: string,
+    period: string,
+    query: string
+) {
+    const eventQueryString = `/analytics/events/query/${programId}.json?dimension=pe:${period}&dimension=ou:${orgUnitIds}&stage=${programStageId}${query}&pageSize=100000`;
```

Comment:
We can use [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/URLSearchParams), but perfectly fine as it is.